data_collection/gazette/spiders/sp_americana.py

```diff
@@ -0,0 +1,68 @@
+import dateparser
+from scrapy import Spider
+import datetime
+from dateutil.relativedelta import relativedelta
+from gazette.items import Gazette
+import re
+
+
+class SpAmericanaSpider(Spider):
+    TERRITORY_ID = "3501608"
+    name = "sp_americana"
+    allowed_domains = ["diariooficial.americana.sp.gov.br"]
+    start_date = datetime.date(2018, 9, 1)  # First gazette available
+    start_urls = ["https://diariooficial.americana.sp.gov.br/diario-oficial-edicaoAnterior.php"]
+    extra_editions = "https://diariooficial.americana.sp.gov.br/diario-oficial-edicaoExtra.php"
+
+    locations = {
+        "gazette": "//*[@class='day table-light']/strong/a",
+        "extra_gazette_url": "//*[@class='list-group-item']/a/@href",
+        "extra_gazette_date": "//*[@class='list-group-item']/text()"}
```

Comment:
**issue** All code must be [correctly formatted](https://github.com/okfn-brasil/querido-diario/blob/main/CONTRIBUTING.md#automated-code-formatting) using Black. If you configure your [development environment](https://github.com/okfn-brasil/querido-diario#development-environment) with the `pre-commit` enabled, this will be applied automatically before you commit anything.

Reply:
Oh, I did not see about the Black, sorry. Too nice that! I was using Vim, only with pure PEP